### ISSUE TITLE
WASM warning: WEBGL_debug_renderer_info is deprecated in Firefox and will be removed. Please use RENDERER.

### ISSUE THREAD

**Issue by bbarker** [bbarker/bracket-lib#5]
The warning you're seeing is related to the `WEBGL_debug_renderer_info` extension in WebGL. This extension provides debug information about the WebGL renderer, but it's being deprecated in Firefox in favor of simply using `RENDERER`.

Here's a breakdown of the warning:

1. **What is `WEBGL_debug_renderer_info`?**  
   It's a WebGL extension that provides more detailed information about the graphics renderer, specifically the `UNMASKED_VENDOR_WEBGL` and `UNMASKED_RENDERER_WEBGL` parameters. These parameters allow developers to retrieve the vendor and renderer of the GPU, which can be useful for debugging or optimizing WebGL applications for specific hardware.

2. **Why is it being deprecated?**  
   Exposing detailed information about the user's GPU can be a privacy concern, as it can be used for fingerprinting users. Browsers are moving towards providing less specific information to protect user privacy.

3. **What should you do?**  
   If your game or application is using `WEBGL_debug_renderer_info` to get the `UNMASKED_VENDOR_WEBGL` and `UNMASKED_RENDERER_WEBGL` parameters, you should consider removing that usage or replacing it with alternative methods. The warning suggests using `RENDERER`, which provides less specific information but is not considered a privacy risk.

   For example, instead of:
   ```javascript
   var gl = canvas.getContext("webgl");
   var ext = gl.getExtension('WEBGL_debug_renderer_info');
   var renderer = gl.getParameter(ext.UNMASKED_RENDERER_WEBGL);
   ```
   You can use:
   ```javascript
   var gl = canvas.getContext("webgl");
   var renderer = gl.getParameter(gl.RENDERER);
   ```

4. **Impact on your game**:  
   If you're not using the detailed renderer information for any critical functionality, the impact should be minimal. However, if you were using it for optimizations or workarounds specific to certain GPUs, you might need to find alternative methods to achieve the same results.

In summary, the warning is informing you of a change in how Firefox handles WebGL debug information. It's a good idea to address this warning to ensure future compatibility and to respect user privacy.
